src/intTest/java/com/surrealdb/driver/SurrealDriverSpecialOperationsTest.java

```diff
@@ -29,93 +23,92 @@ public class SurrealDriverSpecialOperationsTest extends BaseIntegrationTest {
 
     @BeforeEach
     public void setup() {
-        SurrealWebSocketConnection connection = new SurrealWebSocketConnection(testHost, testPort, false);
+        final SurrealWebSocketConnection connection =
+                new SurrealWebSocketConnection(testHost, testPort, false);
         connection.connect(5);
-        driver = new SyncSurrealDriver(connection);
+        this.driver = new SyncSurrealDriver(connection);
     }
 
     @Test
     public void testSignIn() {
-        driver.signIn(TestUtils.getUsername(), TestUtils.getPassword());
+        this.driver.signIn(TestUtils.getUsername(), TestUtils.getPassword());
```

Comment:
We don't need to constantly specify `this`. Please revert - I would like `this` to only be used in ambigous situations. There shouldnt be a driver variable or parameter in this code and if there is, it should have a different name.

Reply:
Removed all the `.this`.